exec/java-exec/src/main/java/org/apache/drill/exec/expr/ClassGenerator.java

```diff
@@ -218,11 +220,16 @@ public HoldingContainer addExpr(LogicalExpression ex) {
   }
 
   public HoldingContainer addExpr(LogicalExpression ex, boolean rotate) {
+    return addExpr(ex, rotate, false);
```

Comment:
I actually think I will just remove the boolean flag altogether. Originally I wanted to limit the scope, but I think it makes sense to always have it enabled.